tools/circt-test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(LLVM_LINK_COMPONENTS Support)
+get_property(dialect_libs GLOBAL PROPERTY CIRCT_DIALECT_LIBS)
+
+add_circt_tool(circt-test circt-test.cpp)
+target_link_libraries(circt-test
+  PRIVATE
+  ${dialect_libs}
+  MLIRBytecodeReader
+  MLIRBytecodeWriter
```

Comment:
Bytecode writer looks unnecessary for what's currently implemented?

Do we need to specify the non-CIRCT dialects here-- LLVM, SCF, so on (those registered explicitly)?

Reply:
Oh yeah excellent point. Those probably got pulled into my build transitively. I'll make these explicit!